### PR TITLE
MD-4848: Added parameters overriding via maven CLI arguments

### PR DESCRIPTION
**Pull request by denysord88**
@agozhiy could you please review this PR?

### CODE REVIEW

File: src/test/java/org/apache/drillui/test/framework/initial/TestProperties.java [hunk 1]
```diff
@@ -22,6 +22,8 @@
 public abstract class TestProperties {
   public static String drillHost;
 
+  public static String OS = getOSType();
+
```
Comment (agozhiy) [inline src/test/java/org/apache/drillui/test/framework/initial/TestProperties.java]: Please remove newlines between the properties

File: src/test/java/org/apache/drillui/test/framework/initial/TestProperties.java [hunk 2]
```diff
@@ -38,15 +40,30 @@
...
-      driverType = WebBrowser.DRIVER.valueOf(p.getProperty("DRIVER_TYPE"));
-      webdriversPath = p.getProperty("WEBDRIVERS_PATH");
-      secureDrill = Boolean.parseBoolean(p.getProperty("SECURE_DRILL"));
-      defaultTimeout = Integer.parseInt(p.getProperty("DEFAULT_TIMEOUT"));
-      drillUserName = p.getProperty("DRILL_USER_NAME");
-      drillUserPassword = p.getProperty("DRILL_USER_PASSWORD");
+      drillHost = loadParameter(p, "DRILL_HOST");
+      driverType = WebBrowser.DRIVER.valueOf(loadParameter(p, "DRIVER_TYPE"));
+      webdriversPath = "webdrivers/" + OS + "_" + driverType;
+      if(OS.equals("WINDOWS")) webdriversPath += ".exe";
```
Comment (agozhiy) [inline src/test/java/org/apache/drillui/test/framework/initial/TestProperties.java]: Should be moved to a separate method.

File: src/test/java/org/apache/drillui/test/framework/initial/TestProperties.java [hunk 3]
```diff
@@ -38,15 +40,30 @@
...
     } catch (Exception e) {
       throw new RuntimeException(e);
     }
   }
+
+  private static String getOSType() {
+    String os = System.getProperty("os.name").toLowerCase();
+    if(os.contains("win")) return "WINDOWS";
+    if(os.contains("mac")) return "MACOS";
+    if(os.contains("nix") || os.contains("nux") || os.contains("aix")) return "LINUX";
```
Comment (agozhiy) [inline src/test/java/org/apache/drillui/test/framework/initial/TestProperties.java]: Change to the if ... else statements. And please use braces.

File: src/test/java/org/apache/drillui/test/framework/initial/TestProperties.java [hunk 4]
```diff
@@ -38,15 +40,30 @@
...
+    return null;
+  }
+
+  private static String loadParameter(Properties initFile, String parameterName) {
+    String property = System.getProperty(parameterName);
+    if(property == null) property = initFile.getProperty(parameterName);
```
Comment (agozhiy) [inline src/test/java/org/apache/drillui/test/framework/initial/TestProperties.java]: Braces.

**Review by agozhiy — CHANGES_REQUESTED** [review]

**Comment by denysord88** [comment]
@agozhiy could you please review changes by your comments?

**Review by agozhiy — APPROVED** [review]

**Comment by agozhiy** [comment]
+1

**Pull request merged**